### ISSUE TITLE
Pictures are not the most recent ones

### ISSUE THREAD

**Issue by ThomasKientz** [Collaborne/cordova-gallery-access#6]
Hi !

Most of the time, depending on the mobile **AND**  sometimes after some days/months of utilisation, pictures are not the most recent ones (chronologically sorted) but appeared to be retrieved kind of randomly from the past (but in order).

I am guessing it's more about the cordova plugin, but do you have any ideas what can cause this ? Or have ever experienced this ?

I am doing :
```
cordovaGallery.load({ count: 3 })
```

**Comment by ankon** [comment]
I don't think we have ever noticed that ... will keep an eye out now and see whether this also happens for us.

**Comment by ThomasKientz** [comment]
To add some infos 

It's happening for a lot of my users on [iOS](https://itunes.apple.com/us/app/boomerang-mail-myself/id1154427984?mt=8) et [Android](https://play.google.com/store/apps/details?id=com.boomerang.app&hl=en), and for a couple of months at least (I was thinking it was very isolated then I get more and more feedbacks with this issue, then it starts happening with my own devices)

Is there any way I can try to debug this ?

Should I use `albumType: 'PHAssetCollectionSubtypeSmartAlbumUserLibrary'`

What are the "albumType" available ?

Thanks

**Comment by ankon** [comment]
This should be the available album types, from a cursory reading of that code: https://github.com/Collaborne/cordova-gallery-api/blob/master/src/ios/GalleryAPI.m#L405

Note that we're using this fork of https://github.com/SuryaL/cordova-gallery-api due to some (unmerged) bug fixes, but questions about the API should probably be directed there.
